tests/cli/test_deploy.py

```diff
@@ -5815,6 +5818,7 @@ async def test_yaml_file_trigger(self, docker_work_pool):
 
             trigger_spec = {
                 "enabled": True,
+                "description": "This is a test trigger",
```

Comment:
These only test the description is included in the client-side payload sent to the server, they don't test that the server receives and stores the description; maybe adding a non-trivial description field to this test would do the trick: https://github.com/PrefectHQ/prefect/blob/main/tests/events/server/models/test_automations.py#L176